### ISSUE TITLE
Add a link to the guided tutorials to GRC's Help menu

### ISSUE THREAD

**Issue by marcusmueller** [gnuradio/gnuradio#5527]
### Feature Description

GRC has a Help menu, but we don't link to our Guided Tutorials there, although they are what new users should read.

As briefly [discussed](https://matrix.to/#/!BMFENoeifGOSJaeGco:gnuradio.org/$LpGysn8roqOp3zGRZlU2RjC-eoOAC4DDblhs0xfsvFQ?via=gnuradio.org&via=matrix.org&via=ei8fdb.org) with @ei8fdb, let's change that:

* Add a link to https://tutorials.gnuradio.org to the help menu, call it "Guided Tutorials"  

### Feature Urgency

medium (would be nice to have in the near future)

**Comment by prsndp** [comment]
@marcusmueller, Hey I would like to work on this issue. Thanks:)

**Comment by ei8fdb** [comment]
@marcusmueller @prsndp I'd like to put together a simple UI design for this. Do I have time? It would be very simple - just text, some http links. thats it.

**Comment by ei8fdb** [comment]
You asked for asap, so I give you asap. 😆

I'd like to propose this (hopefully easy to implement) design below.

https://user-images.githubusercontent.com/1899506/152834904-115cfbd7-2828-44c7-9288-43484c3c07f5.mp4

Before I do anything else, I'd like to get @marcusmueller and @prsndp feedback.

**Comment by marcusmueller** [comment]
> feedback

you're awesome!

**Comment by ei8fdb** [comment]
> > feedback
> 
> you're awesome!

Hah, no not about me, about the design. 🤣 Possible to implement this time?

**Comment by marcusmueller** [comment]
# Feedback

* The mockup is extremely helpful!
* the text restructuring in "Get Help": absolutely. Let's do this right away.
* Same for contacting humans
* All in all, I'm very much for adopting this – @prsndp would that sound feasible to you?

One thing that is more of an impression:

* It's very text-focused (which is logical, the tutorials are as well, and "help" in the non-video way is going to be textual). Would it make sense to add some kind of ideogram for people that have a hard time comprehending English texts¹? <br> That way, we could highlight the way for the user who feels lost, much like role-playing computer games tend to highlight points of relevance.


---
¹ Remark on comprehending longer English texts: much better simply through the addition of paragraphing, it's what I constantly find myself doing when working with posts on StackOverflow and similar sites.<br>
Even if I think my English is quite OK for technical comprehension purposes, I find that many – if not most people – underestimate the effect that transporting one's own structuring of thought by means of text structure has.

**Comment by prsndp** [comment]
Hello, @marcusmueller ! this UI design is extremely well and also helpful for users. So should we start working on it right?

**Comment by ei8fdb** [comment]
> # Feedback
> 
> * The mockup is extremely helpful!
> * All in all, I'm very much for adopting this – 

Good news! 👍

> * It's very text-focused (which is logical, the tutorials are as well, and "help" in the non-video way is going to be textual).

When you say it, you mean the Help window, right? We can improve things later?

> Would it make sense to add some kind of ideogram for people that have a hard time comprehending English texts¹?
> That way, we could highlight the way for the user who feels lost, much like role-playing computer games tend to highlight points of relevance.


Can you describe what the ideogram would contain? If you have an example that'd do also. As I don't fully understand.

> ¹ Remark on comprehending longer English texts: much better simply through the addition of paragraphing, it's what I constantly find myself doing when working with posts on StackOverflow and similar sites.

I understand. Makes sense. I can add some paragraphing to the text as you mention here. I was conscious about  minimising the changes suggested.

**Comment by marcusmueller** [comment]
> Can you describe what the ideogram would contain? If you have an example that'd do also. As I don't fully understand.

coming up!

> When you say it, you mean the Help window, right? We can improve things later?

yes!

> I understand. Makes sense. I can add some paragraphing to the text as you mention here. I was conscious about minimising the changes suggested.

My footnote was more of a praise of how you've already structured things!

**Comment by marcusmueller** [comment]
> If you have an example that'd do also. 

![ideograph guided tutorials](https://user-images.githubusercontent.com/958972/152871371-18687467-0d2d-4ce0-a3ab-67ca4671a1c2.svg)

maybe used like this:

![general_idea_iconographs](https://user-images.githubusercontent.com/958972/152872845-c32deab9-5c2a-43cc-ac9b-92f00ba01704.png)

**Comment by ei8fdb** [comment]
> coming up! (I'm thinking of an icon that conveys the idea of "this is where you go if you need to learn, step by step, how you do it")

Ah understood. I'll try and add that to the design and update there mock-up.

I'm unsure where the iconography used in GRC comes from. I'll propose icons as simple as possible. 

> My footnote was more of a praise of how you've already structured things!
 
Ah ok! I'll take that. 🙂 I'll take a look again to see if it can be still improved.

**Comment by ei8fdb** [comment]
@marcusmueller @prsndp Here you go, I've updated the designs:

## Wireframes
1. GRC default main window
https://share.balsamiq.com/c/uwVFmFnFwsvY9aQSjYv32i.png

2. User clicks on Help in file menu
https://share.balsamiq.com/c/jrQ9UzEaRWP2XY6cT2xFPq.png

3. User clicks on Get help
https://share.balsamiq.com/c/2nfkKsGBgq3msakTSYsy9g.png
 
## Changes

1. Ideograms

I've updated my mockups with the ideograms as Marcus suggested. I've used 3 icons from the free [Font Awesome icon gallery](I've used 3 icons from the free Font Awesome icon gallery):

- [keyboard](https://github.com/FortAwesome/Font-Awesome/blob/master/js-packages/@fortawesome/fontawesome-free/svgs/regular/keyboard.svg)
- [book](https://github.com/FortAwesome/Font-Awesome/blob/master/svgs/solid/book.svg)
- [comments](https://github.com/FortAwesome/Font-Awesome/blob/master/svgs/solid/comments.svg)

@marcusmueller can you tell me if these are available in the GRC UI framework?

2. Updated labels

I've also changed the labels for each section of Usage tips, Tutorials, and Support.

**Comment by Anjali503** [comment]
Hi, I’m a new contributor to GNU Radio.

I’d like to work on this issue if it’s still available. 
Please let me know if there’s anything specific I should be aware of before starting.

**Comment by marcusmueller** [comment]
@Anjali503 Yes, please go ahead. I think ei8fb's mockups are still immensely valuable, and him linking to the right icons/ideograms is basically solving the hard part that's not writing code.

@Anjali503 please prioritize fixing the Qt version of GRC (you can launch it via `gnuradio-companion --qt`). Open a PR as soon as you have the functionality implemented, and include a screenshot of the new help dialog, please!

**Comment by Gowri213** [comment]
@marcusmueller Thanks for the clarification! 
I'm also working on this and will follow the Qt GRC setup and include a screenshot of the Help menu with the guided tutorials link.

**Comment by marcusmueller** [comment]
Dear @Gowri213,
 @Anjali503 has just yesterday said they want to work on this. There's enough other similar UI issues! Please pick one, it's not great if I have to pick between two near-identical solutions.

**Comment by Anjali503** [comment]
Thanks for the clarification, @marcusmueller!

I’ll prioritize the Qt version of GRC and use ei8fdb’s mockups/icons as reference.
I’ll open a PR with a screenshot once the functionality is implemented.

**Comment by Gowri213** [comment]
@marcusmueller Thanks for the update! I'm happy to pick a different UI issue. I'll look for another to contribute!

**Comment by Anjali503** [comment]
Hi @marcusmueller, quick update: I’ve implemented the Guided Tutorials link in the Qt GRC Help menu and opened a PR here:

https://github.com/gnuradio/gnuradio/pull/7991

Includes the requested functionality and follows existing Qt GRC patterns. Thanks!
